docs/json_schemas/dns_record/v0/requirer.json

```diff
@@ -0,0 +1,261 @@
+{
+  "$defs": {
+    "BaseModel": {
+      "properties": {},
+      "title": "BaseModel",
+      "type": "object"
+    },
+    "DnsRecordRequirer": {
+      "description": "List of domains for the provider to manage.",
+      "properties": {
+        "dns_domains": {
+          "anyOf": [
+            {
+              "items": {
+                "$ref": "#/$defs/RequirerDomains"
+              },
+              "type": "array"
+            },
+            {
+              "type": "null"
+            }
+          ],
+          "description": "List of domains for the provider to manage.",
+          "title": "Dns Domains"
+        },
+        "dns_entries": {
+          "anyOf": [
+            {
+              "items": {
+                "$ref": "#/$defs/RequirerEntries"
+              },
+              "type": "array"
+            },
+            {
+              "type": "null"
+            }
+          ],
+          "description": "List of DNS records for the provider to manage.",
+          "title": "Dns Entries"
+        }
+      },
+      "required": [
+        "dns_domains",
+        "dns_entries"
+      ],
+      "title": "DnsRecordRequirer",
+      "type": "object"
+    },
+    "RecordClass": {
+      "description": "Represent the DNS record classes.",
+      "enum": [
+        "IN",
+        "CH",
+        "HS"
```

Comment:
Classes other than `IN` (`CH` and `HS`) are pretty obsolete... I'm wondering if we should support them. Still defining the `record_class` is needed to support potential futures extensions, but currently the only valid one is `IN` I think.

Reply:
Let's drop support, then